src/main/java/net/mcreator/element/types/interfaces/IPOIProvider.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.types.interfaces;
+
+import net.mcreator.element.parts.MItemBlock;
+
+import java.util.List;
+
+public interface IPOIProvider {
+
+	/**
+	 * This method determines what POI blocks as MItemBlock list are provided by this generatable element
+	 *
+	 * @return A list of MItemBlock that are POIs provided by this generatable element

```

Comment:
_POIs_ is not abbreviated **Point of interests**, the main noun is not the last word in this collocation. This should be just **POI** IMO.

Reply:
Done!